### PR TITLE
DOCS-330 Updated transfer criteria interface documentation

### PR DESCRIPTION
**Pull request by danetsao**
Per [DOCS-330](https://issues.folio.org/browse/DOCS-330), updated transfer criteria documentation with up to date information for Bama's updates to the bursar export feature.

PR to merge into branch `main` of `ualibweb/folio-docs` for feedback, then will create pr to `folio-org/docs` . 

Latest snapshot of docs is at the bottom of the page.

### CODE REVIEW

**Comment by danetsao** [comment]
Here is a preview of the markdown, for readability.

Let me know what you guys think, if this is comprehensive enough or more is needed in some parts.

Screenshot 2023-07-10 at 4 09 34 PM

**Comment by enettifee** [comment]
Hi Dane - thanks for sharing this with me and asking for feedback! Some thoughts:

- I think a good approach would be to have a section for most of the accordions in the interface, and then document that way - I think something like Scheduling, Criteria, Aggregate by Patron, Formats, Preview, and Transfer accounts to would make sense to an end user. That also is fairly close to the style we usually follow since it kind of follows functionality down the page, if that makes sense.
- You could also document each of the three formats accordions separately, but it looks like a lot of overlapping functionality, so I think doing it in one section also makes sense.
- I think the documentation for the formatting blocks will benefit from some conceptual explanation of what it's trying to do - e.g., with the format accordions, E.g., what I think is happening is that you are constructing a string out of different blocks, maybe block 1 is a word, block 2 is a space, block 3 is a date, block 4 is another space, and block 5 is another word. I can click "add" if I need more blocks, etc. etc. Especially I think it will be helpful to connect that conceptual explanation to the preview accordion - "You can build a string with the blocks, then scroll down to the preview accordion to see what it would look like", etc etc.
- Wherever accordions are used, SMEs will expect definitions. A bulleted list is fine, it can be very simple, but people will want definitions. (And I realize this is not a development request, but "Arbitrary text" is odd and will translate weird to  other languages. I would just say "Text" or even "Word" or "Text String", depending on what you actually are expecting people to put in there)
- Examples will definitely help - it's going to be hard to do without images, and hard because the boxes aren't numbered or labeled - but maybe something like 

> "To build the header 
> Batch Test 2	2023711DU LIBRARY 3TXT_CNTDU LIBRARIES DEBITS
> your header format would have 8 blocks, that would be configured like this:
> * Block 1 - Arbitrary Text, with a value of "Batch Test 2"
> * Block 2, - Tab
> * Block 3 - Current Date , with a format of "Month"
> *......."

etc etc. Pictures would make it much easier, but docs.folio.org just isn't there yet.

File: content/en/docs/Settings/Settings_users/Settings_users.md [hunk 1]
```diff
@@ -364,16 +364,21 @@ Use this setting to configure the criteria for transferring accounts. For more i
...
-Select the transfer owner from the **Transfer owner** drop-down list.
-Select a transfer account from the **Transfer account** drop-down list.
-Select the **Fee/fine owner**.
-Add **Transfer type**, **Transfer description**, and **Transfer code** to each **Fee/fine type**.
-Click **Save** to save the transfer criteria or click **Run manually** to run a report. 
+1. In the **Transfer criteria** pane, go to the **Scheduling** dropdown and select the desired scheduling period.
+2. In the **Criteria** dropdown, select the criteria for the transfer and fill in the metadata.
+3. In the **Aggregate by patron** dropdown, check the box if you want to aggregate by patron.
+4. In the **Header format** dropdown, configure your header format.
+5. In the **Account data format** dropdown, configure the body of the file.
```
Comment (ncovercash) [inline content/en/docs/Settings/Settings_users/Settings_users.md]: Keep in mind that this can be called `Aggregate data format`, too.

**Review by ncovercash — CHANGES_REQUESTED** [review]
It might be good to add some examples, too!  You can see [what we/Erin did for calendar](https://docs.folio.org/docs/settings/settings_calendar/settings_calendar/), where it has a lot of information about how to perform common actions, examples that can demonstrate a lot of functionality (chinese new year), etc

**Comment by danetsao** [comment]
Current screenshot

- Updated structure to give each section its own section in docs
- Working on adding options for each section and examples for some
- Stilll have to add rest of options to **Criteria** and **Transfer account**

Screenshot 2023-07-12 at 10 18 18 AM

File: content/en/docs/Settings/Settings_users/Settings_users.md (hunk 1 above)
Comment (ncovercash) [reply content/en/docs/Settings/Settings_users/Settings_users.md]: @danetsao the "Account data format" is renamed "Aggregate data format" when the aggregate mode is enabled

Comment (danetsao) [reply content/en/docs/Settings/Settings_users/Settings_users.md]: @ncovercash Gotcha, that makes sense, Ill note that.

Comment (nhanaa) [reply content/en/docs/Settings/Settings_users/Settings_users.md]: Actually it's Patron data format, but yeah it does change when aggregate mode is enabled.

**Comment by danetsao** [comment]
Should have every section covered with mostly all available options. Also added an example to sections **Criteria** and **Aggregate by patron**.

Preview here for readability:

Screenshot 2023-07-17 at 12 42 44 PM
Screenshot 2023-07-17 at 12 42 27 PM

**Review by nhanaa — APPROVED** [review]

**Review by nhanaa — APPROVED** [review]

**Pull request merged**